src/stdatamodels/jwst/datamodels/photom.py

```diff
@@ -101,6 +101,23 @@ class MirMrsPhotomModel(ReferenceFileModel):
     pixsiz : numpy array
         An array-like object containing pixel-by-pixel size values, in units of
         square arcseconds (arcsec^2).
+
+   timecoeff_ch1: numpy table
```

Comment:
All of these new attribute name entries need to be indented 1 more space to line up with all previous entries. Also need a whitespace before the colon in each entry.